[docs] Why are version rangers not allowed? (#8831)
conan-io/conan-center-index: docs/faqs.md

  * [What is the policy for supported python versions?](#what-is-the-policy-for-supported-python-versions)
  * [How to package libraries that depend on proprietary closed-source libraries?](#how-to-package-libraries-that-depend-on-proprietary-closed-source-libraries)
  * [How to _protect_ my project from breaking changes in recipes?](#how-to-_protect_-my-project-from-breaking-changes-in-recipes)<!-- endToc -->

>   * [Why are version rangers not allowed?](#why-are-version-ranges-not-allowed)<!-- endToc -->

Reviewer: ```suggestion
  * [Why are version ranges not allowed?](#why-are-version-ranges-not-allowed)<!-- endToc -->
```

   and lockfiles. 

Keep reading in the [consuming recipes section](consuming_recipes.md).

> ## Why are version rangers not allowed?

Reviewer: ```suggestion
## Why are version ranges not allowed?
```

> Version ranges are a useful Conan feature, find the documentation [here](https://docs.conan.io/en/latest/versioning/version_ranges.html). However in the context of ConanCenter they pose a few key challenges, most notably "Build Reproducibility".

Reviewer: ```suggestion
Version ranges are a useful Conan feature, find the documentation [here](https://docs.conan.io/en/latest/versioning/version_ranges.html). However, in the context of ConanCenter they pose a few key challenges, most notably "Build Reproducibility".
```

> Version ranges are a usefull Conan feature, find the documentation [here](https://docs.conan.io/en/latest/versioning/version_ranges.html). However in the context of ConanCenter they pose a few key challanges, most notably "Build Reproducibility".

Reviewer: ```suggestion
Version ranges are a useful Conan feature, find the documentation [here](https://docs.conan.io/en/latest/versioning/version_ranges.html). However in the context of ConanCenter they pose a few key challenges, most notably "Build Reproducibility".
```

> If consumers tries to download and build the recipe at a later time, it may resolve to a different package version that may not be compatable. In order to 
> prevent these types of issues we have decided to only allow exact requirements versions.

Reviewer: ```suggestion
If consumers try to download and build the recipe at a later time, it may resolve to a different package version that may not be compatible. In order to prevent these types of issues, we have decided to only allow exact requirements versions.
```
